CodingChallenge/Views/SearchResultCell.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SearchResultCell.swift
+//  CodingChallenge
+//
+//  Created by Edgar Barocio on 6/23/19.
+//  Copyright © 2019 Edgar Barocio. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultCell: UICollectionViewCell {
+
+    @IBOutlet weak var albumArt: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var artistLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        //update(with: nil, title: nil, artist: nil)
+    }
+    
+    func update(with imageURL: String?, title: String?, artist: String?) {
+        
+        let url = URL(string: imageURL ?? "")
```

Comment:
I'd use some kind of if-let here to avoid using the force unwraps below if possible. Handling errors and nil cases looks good in these kinds of projects, even if it's just with a print.